index.shtml

```diff
@@ -113,16 +113,22 @@
 
       <hr/><br/>
 
-      <b>Find us on social media!</b>
-
-      <table>
-          <tr>
-              <td><a href="https://www.facebook.com/groups/genesis.scifi" target="_blank"><img src="images/facebook-logo.svg" alt="Facebook" width="40" height="40"/></a></td>
-              <td><a href="https://twitter.com/Genesis_SciFi" target="_blank"><img src="images/twitter-logo-250x250.png" alt="Twitter" width="40" height="40"/></a></td>
-              <td><a href="https://instagram.com/Genesis_SciFi" target="_blank"><img src="images/instagram-logo-250x250.png" alt="Instagram" width="40" height="40"/></a></td>
-          </tr>
-      </table>
-
+	  <table class="links">
+	    <tr>
+	      <th>Find us on social media!</th>
+	      <th>Donate!</th>
+	    </tr>
+	    <tr>
+	      <td>
+                <a href="https://www.facebook.com/groups/genesis.scifi" target="_blank"><img src="images/facebook-logo.svg" alt="Facebook" width="40" height="40"/></a>
+                <a href="https://twitter.com/Genesis_SciFi" target="_blank"><img src="images/twitter-logo-250x250.png" alt="Twitter" width="40" height="40"/></a>
+                <a href="https://instagram.com/Genesis_SciFi" target="_blank"><img src="images/instagram-logo-250x250.png" alt="Instagram" width="40" height="40"/></a>
+              </td>
+	      <td>
+                <a href="https://www.buymeacoffee.com/genesissf" target="_blank"><img src="https://cdn.buymeacoffee.com/buttons/v2/arial-blue.png" alt="Buy Me A Coffee" style="height: 60px !important;width: 217px !important;" ></a>
```

Comment:
I was just playing around with sizing, and I think you should make the heights of the social media icons the same as that of the button. I've played around with things, and making the button to be 40x145 feels a bit too small. Similarly, making the buttons 60x60, while okay, feels just a tad too large. I opted for making the heights 50 all around, and the button width 181. Thought it does make the "Buy me a coffee" text a little blurry.

```suggestion
                <a href="https://www.buymeacoffee.com/genesissf" target="_blank"><img src="https://cdn.buymeacoffee.com/buttons/v2/arial-blue.png" alt="Buy Me A Coffee" style="height: 50px !important;width: 181px !important;" ></a>
```

Either way, the core idea is to make the button and icons all the same height.